ros_buildfarm/templates/devel/devel_create_tasks.Dockerfile.em

```diff
@@ -18,6 +18,11 @@ ENV DEBIAN_FRONTEND noninteractive
 
 RUN useradd -u @uid -m buildfarm
 
+# Todo: check to only include this in the case of gpu jobs
```

Comment:
although this snippet can probably stay in any docker image (even those without GPU support) the elegant way would be to make it dependent of a required or present GPU support. I'm not sure how to do it in empy or if we need another template file.

Reply:
I think I found the way 0f4ddc2 (bad variable name corrected in a second commit).

Reply:
> with that in mind, does this Dockerfile need this setup at all or is it something that should be added to the task-performing dockerfiles rather than the task-creating one?

Given that the commands needs to be added in the dockerfile I understand that the task managing the creation of them is the one that should take care of this action.